contracts/utils/deploy-l2.js

```diff
@@ -112,7 +113,8 @@ function deployOnBase(opts, fn) {
 
   return main;
 }
-function deployOnBaseWithGuardian(opts, fn) {
+
+function deployOnL2WithMultisig(opts, fn, forkSignerAddress) {
```

Comment:
I think multisig is too ambiguous. Does it mean the Admin (5/8), Guardian (2/8) or the new multi-chain Admin (5/8)?